queries/highlights.scm

```diff
@@ -1,5 +1,9 @@
 ; Function calls
 
+(call_expression
+  function: (identifier) @function.builtin
+    (.match? @function.builtin "append|copy|delete|len|cap|make|new|complex|real|imag|close|panic|recover|print|println"))
```

Comment:
Based on testing in Emacs with the `tree-sitter-query-builder` command, I believe this regexp needs a minor tweak. As is, for me it is matching most calls to top-level functions.

The following change seems to resolve it for me:

```suggestion
    (.match? @function.builtin "^(append|copy|delete|len|cap|make|new|complex|real|imag|close|panic|recover|print|println)$")
```